packages/@react-aria/toast/test/useToast.test.js

```diff
@@ -64,10 +65,11 @@ describe('useToast', () => {
   });
 
   it('handles onClose', function () {
-    let {closeButtonProps} = renderToastHook({onClose});
+    let {closeButtonProps} = renderToastHook({onClose, onRemove});
     closeButtonProps.onPress();
 
     expect(onClose).toHaveBeenCalledTimes(1);
+    expect(onRemove).toHaveBeenCalledTimes(1);
```

Comment:
Might be worthwhile adding a check that `onRemove` is called with the provided `idKey`